### ISSUE TITLE
Octopull keeps on giving the 'Authentication error' warning

### ISSUE THREAD

**Issue by daveystruijk** [rmhartog/octopull#12]
In production, whenever I try to view a pull request with octopull, I get the following message at the top of my page:

```
Authentication error
You are not logged in to Octopull. (Log in)
```

If I press the Login button, it doesn't log me in and still gives this warning every time.

My browser also gives me a 403 error on trying to fetch the pull request:

```
Failed to load resource: the server responded with a status of 403 (Forbidden) https://octopull.rmhartog.me/api/repos/rmhartog/octopull/pulls/7/diff/aeb6471b669d3ffc03a19c9b49c3f010e1bd5542/f5b15fb2c196503d645b61cbff2ed1aa826d12d6
octopull.js:17718
```

**Comment by daveystruijk** [comment]
~~Seems to be working again somehow! Did you change anything?~~ Sometimes it works, sometimes it doesn't. :/

**Comment by rutgervandenberg** [comment]
I previously had this problem, back then I 'fixed' it by allowing octopull cookie. Now it doesn't seem to work at all. A number of people I know have the same problem as well.

**Comment by rmhartog** [comment]
Sorry for the delay, I was out of the country, but I'm looking into this. I'll report back here.

**Comment by jnieuwdorp** [comment]
Any progress? The login messages are starting to stack up
![Screenshot](https://cloud.githubusercontent.com/assets/5521876/7723277/bfd821c4-fee7-11e4-9542-4d6a83d895ab.png)

**Comment by rmhartog** [comment]
I'm still looking into this, the issue remains vague and varies between users. The messages should not be able to stack, I'll look into that immediately. In the meantime, could you try to verify the steps in [this comment](https://github.com/rmhartog/octopull/issues/2#issuecomment-94707199)? Let me know if you need more elaborate instructions on how to do so.

**Comment by jnieuwdorp** [comment]
That is indeed what happens. The switch to the secure socet seems to function normal.

**Comment by rmhartog** [comment]
The plugin has been updated to stop the stacking from happening, the update should propagate through the chrome store soon. Did the request that has the 403 error response include the `github_token` cookie? That is very strange.

**Comment by rmhartog** [comment]
Please see if the problem has been resolved now and comment on #14
